rest_framework_swagger/introspectors.py

```diff
@@ -266,7 +260,7 @@ def get_description(self):
         if self.yaml_parser.get_param('replace_docs', False):
             docstring = method_docs
         else:
-            docstring = "\n\n".join(filter(None, [class_docs, method_docs]))
+            docstring = "\n\n".join([_f for _f in [class_docs, method_docs] if _f])
```

Comment:
im guessing this was auto-generated. but something nicer than _f would be desirable.